iree/compiler/Conversion/CodegenUtils/WorkgroupCalculation.h

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google LLC
```

Comment:
Same as above. Rename this to GetNumWorkgroups.h

Reply:
Done!